README.md

```diff
@@ -2,20 +2,49 @@
 
 ## Installation
 
-Install [`preCICE`](https://precice.org/installation-overview.html), [`pyMOR`](https://github.com/pymor/pymor#installation-via-pip) and [`deal.II`](https://dealii.org/current/readme.html) (v9.2 or greater). Afterwards, install the [pyMOR-deal.II wrapper](https://github.com/pymor/pymor-deal.II) using
+The setup requires a variety of software packages. Install [`preCICE`](https://precice.org/installation-overview.html) (v2.4.0 or greater), [`pyMOR`](https://github.com/pymor/pymor#installation-via-pip) (at least version [ef3242c](https://github.com/pymor/pymor/commit/ef3242c9aebd9c1046fb6d2b80d414284abca1ad) as set in the `requirements.txt` is required), [`FEniCS`](https://fenicsproject.org/download/archive/) (legacy version as set in the `requirements.txt`) and [`deal.II`](https://dealii.org/current/readme.html) (v9.2 or greater). Afterwards, install the [`pyMOR-deal.II wrapper`](https://github.com/pymor/pymor-deal.II) using
 
-```
+```bash
 git clone --recurse-submodules https://github.com/pymor/pymor-deal.II.git
 python3 -m pip install ./pymor-deal.II
 ```
 and clone this repository using
 
-```
+```bash
 git clone --recurse-submodules https://github.com/DavidSCN/mor-coupling.git
-cmake -B ./mor-couling/build -S ./mor-coupling
-cmake --build ./mor-couling/build 
 ```
+
+The deal.II-based executable can be compiled using
+
+```bash
+cmake -B ./mor-coupling/build -S ./mor-coupling
+cmake --build ./mor-coupling/build
+```
+
+## Running a simulation
+
+The example setup is located in the `example` directory. In a first step, the offline phase, multiple coupled simulations need to be performed in order to generate the parametrized reduced basis later on. By default the coupled simulation is performed `5` times using uniform samples of the diffusion coefficient. Afterwards, `5` random diffusion coefficients are used in order to compare the full order model and the reduced order model. In order to execute all simulations, execute
+
+```bash
+./run.sh -n 15
+```
+
+from the `dirichlet-fenics` directory and
+
+```bash
+python3 heat_equation_reduced.py
+```
+
+from the `neumann-reduced` directory. The Neumann participant prints out statistics regarding the error and speedup of the reduced basis.
```

Comment:
Running these two scripts triggers the offline phase and afterwards the online phase?
Maybe we could make this more explicit.